ci/pipelines/deploy.yml

```diff
@@ -0,0 +1,416 @@
+
+  - name: Staging Deploys
+    jobs:
+      - deploy-adminusers-staging
+      - deploy-cardid-staging
+      - deploy-card-connector-staging
+      - deploy-card-frontend-staging
+      - deploy-directdebit-connector-staging
+      - deploy-directdebit-frontend-staging
+      - deploy-ledger-staging
+      - deploy-products-staging
+      - deploy-products-ui-staging
+      - deploy-publicapi-staging
+      - deploy-publicauth-staging
+      - deploy-selfservice-staging
+      - deploy-notifications-staging
+
+  - name: Staging Smoke Tests
+    jobs:
+      - card-payment-smoke-tests-staging
+      - direct-debit-smoke-tests-staging
+      - products-smoke-test-staging
+
+resource_types:
+  - name: cf-cli
+    type: docker-image
+    source:
+      repository: nulldriver/cf-cli-resource
+
+resources:
+  - name: omnibus
+    type: git
+    icon: github-circle
+    source:
+      uri: https://github.com/alphagov/pay-omnibus
+      branch: develop
+
+  - name: paas-staging
+    type: cf-cli
+    icon: cloud-upload-outline
+    source:
+      api: https://api.london.cloud.service.gov.uk
+      org: govuk-pay
+      space: staging
+      username: ((cf-username))
+      password: ((cf-password))
+
+  - &image
+    name: adminusers
+    type: docker-image
+    icon: docker
+    source: &image-source
+      repository: govukpay/adminusers
+      tag: latest-master
+      username: ((docker-username))
+      password: ((docker-password))
+
+  - <<: *image
+    name: cardid
+    source:
+      <<: *image-source
+      repository: govukpay/cardid
+
+  - <<: *image
+    name: card-connector
+    source:
+      <<: *image-source
+      repository: govukpay/connector
+
+  - <<: *image
+    name: directdebit-connector
+    source:
+      <<: *image-source
+      repository: govukpay/directdebit-connector
+
+  - <<: *image
+    name: directdebit-frontend
+    source:
+      <<: *image-source
+      repository: govukpay/directdebit-frontend
+
+  - <<: *image
+    name: card-frontend
+    source:
+      <<: *image-source
+      repository: govukpay/frontend
+
+  - <<: *image
+    name: ledger
+    source:
+      <<: *image-source
+      repository: govukpay/ledger
+
+  - <<: *image
+    name: products
+    source:
+      <<: *image-source
+      repository: govukpay/products
+
+  - <<: *image
+    name: products-ui
+    source:
+      <<: *image-source
+      repository: govukpay/products-ui
+
+  - <<: *image
+    name: publicapi
+    source:
+      <<: *image-source
+      repository: govukpay/publicapi
+
+  - <<: *image
+    name: publicauth
+    source:
+      <<: *image-source
+      repository: govukpay/publicauth
+
+  - <<: *image
+    name: selfservice
+    source:
+      <<: *image-source
+      repository: govukpay/selfservice
+
+  - <<: *image
+    name: notifications
+    source:
+      <<: *image-source
+      repository: govukpaypaas/notifications
+
+jobs:
+  - name: deploy-adminusers-staging
+    serial_groups: [adminusers-stg]
```

Comment:
Should all deploy jobs be part of the same serial group, as we probably want only one app to be deployed at a time? Or is that handled in a different way?

Reply:
Probably one for the team to discuss as there are cases where various apps can safely be deployed together.